ddtrace/contrib/psycopg/patch.py

```diff
@@ -16,6 +16,14 @@
 
 from ...internal.utils.formats import asbool
 from ...internal.utils.version import parse_version
+from ...propagation._database_monitoring import _DBM_Propagator
+
+
+def _psycopg2_sql_injector(dbm_comment, sql_statement):
```

Comment:
Do we have a way of testing each integration's injector already? If not, should we add one?

Reply:
This injector is tested in the psycopg functional tests. I tried to avoid testing internal functions. This locks us into a specific implementation and doesn't provide a lot of value.